### DISCUSSION TITLE
Event handlers for ScrollBar: MouseEnter/MouseLeave

### DISCUSSION THREAD

**Discussion by mauricio-lomeli-planview** [idiotWu/smooth-scrollbar#533] [Ideas]
I'd like to see an event handler for when the mouse enters the scroll bar. I've tried looking this up in the **issues** category and documentation but couldn't find it.

```tsx
<ScrollBar 
      onMouseEnter={() => do_someting()}
      onMouseLeave={() => do_something()}
</ScrollBar>
```

**Comment by sadeghbarati** [comment]
I managed to register `mouseenter` and `mouseleave` events to the same element as `smooth-scrollbar` inited on

What framework/lib are you using?

**Reply by mauricio-lomeli-planview** [reply]
Nothing yet. I'm trying to find a scroll bar handler package for TypeScript which can handle this functionality. For example, the scroll bar changes on MacOS, windows, firefox, chrome, etc.
